frontend/src/v5/ui/controls/modal/formModal/formDialog.component.tsx

```diff
@@ -29,12 +29,14 @@ import {
 	FormDialogActions,
 	RemoveWhiteCorners,
 	Subtitle,
+	SubmitButton,
 } from './formDialog.styles';
 
 export interface IFormModal extends Omit<DetailedHTMLProps<FormHTMLAttributes<HTMLFormElement>, HTMLFormElement>, 'ref'> {
 	onClickClose?: () => void;
+	onSubmit?: (event) => void;
 	onClickCancel?: () => void;
-	title?: string;
+	title?: any;
```

Comment:
Would it maybe be worth using the TruncateText component in this file, as we probably want all modal title to be truncated if they get too long or the modal gets too thin